src/main/java/org/openstreetmap/atlas/checks/maproulette/data/Task.java

```diff
@@ -58,6 +58,7 @@ public Location getLocation()
     protected static final String TASK_PARENT_ID = "parent";
     protected static final String TASK_TYPE = "type";
     private static final String KEY_DESCRIPTION = "description";
+    private static final String CHECK_GENERATOR = "check";
```

Comment:
I believe this value will be overwritten if we have a feature with [the tag "check"](https://taginfo.openstreetmap.org/keys/check).  I suggest going with `flag:check`, [like the tippecanoe output](https://github.com/osmlab/atlas-checks/blob/dev/src/main/java/org/openstreetmap/atlas/checks/event/CheckFlagEvent.java#L348) or `metadata`: {check: ""}, or something similar

Reply:
We could also add: `flag:generator`: "Atlas Checks"

Reply:
Done!

Reply:
Now ```flag:check``` and ```flag:generator``` are in the properties field for a Task